### DISCUSSION TITLE
netty-all-4.1.69 no io package

### DISCUSSION THREAD

**Discussion by microapplication** [netty/netty#11752] [Q&A] [answered]
in netty-all-4.1.69.Final.jar , no io package, so cannot to be used.

**Comment by normanmaurer** [comment, marked as answer]
@microapplication it depends on all the other modules so your dependency tool should pull in its dependencies. See also:
https://netty.io/news/2021/10/11/4-1-69-Final.html

**Comment by microapplication** [comment]
ok,thanks.
________________________________
发件人: Norman Maurer ***@***.***>
发送时间: 2021年10月12日 0:06
收件人: netty/netty ***@***.***>
抄送: microapplication ***@***.***>; Mention ***@***.***>
主题: Re: [netty/netty] netty-all-4.1.69 no io package (Discussion #11752)


@microapplication<https://github.com/microapplication> it depends on all the other modules so your dependency tool should pull in its dependencies. See also:
https://netty.io/news/2021/10/11/4-1-69-Final.html

―
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/netty/netty/discussions/11752#discussioncomment-1459962>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD6Z6DTOOIKHSWCJWN73NATUGMDQZANCNFSM5FYSCTNA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
